airbyte-integrations/connectors/destination-snowflake/src/main/kotlin/io/airbyte/integrations/destination/snowflake/spec/SnowflakeConfiguration.kt

```diff
@@ -19,7 +19,7 @@ data class SnowflakeConfiguration(
     val authType: AuthTypeConfiguration,
     val cdcDeletionMode: CdcDeletionMode,
     val legacyRawTablesOnly: Boolean?,
-    val internalTableDataset: String,
+    val internalTableSchema: String?,
```

Comment:
This was marked as nullable in the spec, but non-nullable here.  I assume this was a mistake.

Reply:
I see why this was marked as non-null.  The class made sure this value is not null when copying the values from the spec into the config.  The PR has been updated to use the default raw namespace if and only if the use legacy raw tables boolean is true.